setup.py

```diff
@@ -3,19 +3,14 @@
 import sys
 import subprocess as subp
 import platform
-import os
 
 cwd = Path(__file__).parent
 
 sys.path.append(str(cwd))
 from cmake_ext import CMakeExtension, CMakeBuild  # noqa: E402
 
-if not os.environ.get("CI", False):
-    if (cwd / ".git").exists():
-        # make sure that submodules are up-to-date,
-        # it is a common error to forget this when
-        # switching between development branches
-        subp.check_call(["git", "submodule", "update"])
+subp.call(["git", "submodule", "update"])
+
```

Comment:
Please restore the original. When you install from an sdist, you have the source code but it is not a git repo.

Reply:
I see. It will be restored.